module5/src/main/java/Main.java

```diff
@@ -0,0 +1,224 @@
+import bean.Faculty;
+import bean.Group;
+import bean.Student;
+import exceptions.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        List<Faculty> facultyList = createFacultyList();
+        List<Faculty> emptyFacultyList = emptyFacultyList();
+        List<Faculty> facultyWithoutGroup = facultyWithoutGroup();
+        List<Faculty> groupWithoutStudents = groupWithoutStudents();
+        List<Faculty> studentWithoutSubjects = studentWithoutSubjects();
```

Comment:
Now please create List of Lists and do in `for` loop the next:
- validate each list
- calculate each list's average values (all three methods should be called)